homeassistant/components/stt/__init__.py

```diff
@@ -154,5 +383,5 @@ def _metadata_from_header(request: web.Request) -> SpeechMetadata:
             sample_rate=args["sample_rate"],
             channel=args["channel"],
         )
-    except TypeError as err:
+    except ValueError as err:
```

Comment:
I don't see how there can be a TypeError. It's either converting a string to an integer or creating enum instances that can fail, which both will raise ValueError if the input is wrong.